acceptanceTests/Makefile

```diff
@@ -1,2 +1,3 @@
 test: ## Run acceptance tests.
+	@npm install cypress@10.0.1 -y
```

Comment:
can this be done in the setup file?

Reply:
no. it should be installed when we are in the directory of the acceptance test.
so it feels like this place is the best place to put it in

Reply:
(if it is installed before (like I tried) it fails with error of "module not found" when we are running the "npx cypress run.."